MvvmCross.Droid.Support.V7.RecyclerView/MvxRecyclerAdapter.cs

```diff
@@ -242,5 +265,23 @@ public virtual void NotifyDataSetChanged(NotifyCollectionChangedEventArgs e)
                     exception.ToLongString());
             }
         }
+
+        public void ClearAllBindings()
```

Comment:
Can we mark this virtual, so in case of someone inheriting from this Adapter wanting to change the ClearAllBindings behavior, they can do so easily?